DEV/org.openl.core/src/org/openl/binding/impl/MethodNodeBinder.java

```diff
@@ -121,6 +124,19 @@ public IBoundNode bind(ISyntaxNode node, IBindingContext bindingContext) throws
         }
     }
 
+    private void validateMethodParameter(IMethodCaller methodCaller, IBoundNode[] children) {
+        if (methodCaller instanceof CastingMethodCaller) {
+            IOpenCast[] casts = ((CastingMethodCaller) methodCaller).getCasts();
+            if (casts.length == children.length) {
+                for (int i = 0; i < casts.length; i++) {
+                    if (casts[i] != null && children[i] instanceof LiteralBoundNode) {
+                        casts[i].convert(((LiteralBoundNode) children[i]).value);
```

Comment:
Wrap conversion to `try-catch` block to store all errors to the `bindingContext`.

Reply:
Use `bindingContext.addMessage()` and `bindingContext.addError()` for logging warnings and errors.
Be care to log the location of the wrong code using `ISyntaxNode`